packages/@initiative.dev/ide/src/ide/tools/data-flow-inspector/index.tsx

```diff
@@ -7,30 +7,34 @@ import { Canvas } from "./canvas.js";
 const cls = bemClasses("initiative-data-flow-inspector");
 
 export interface DataFlowInspectorProps {
-  controller: CommandController<DialogCommand>;
   document: SceneDocument;
   selectedNode: string | null;
   onSelectedNodeChange(nodeId: string | null): void;
+  className?: string;
+  hidden?: boolean;
 }
 
 export function DataFlowInspector({
-  controller,
+  className,
   document,
   selectedNode,
   onSelectedNodeChange,
+  hidden,
 }: DataFlowInspectorProps) {
   const [zoom, setZoom] = useState(100);
 
   return (
-    <Dialog className={cls.block()} commandStream={controller}>
+    <div
+      className={cls.block(className)}
+      style={{ visibility: hidden ? "hidden" : "visible" }}
```

Comment:
Avoid using the `style` prop if possible. In this case, you can apply the `visibility: hidden` style with a CSS class. Apply a BEM modifier like this:

```ts
<div
  className={cls.block(className, hidden && "hidden")}
/>
```

Then inside `app.scss`, apply the styles to that CSS class:
```scss
.initiative-editor {
  &__data-flow-inspector {
    &--hidden {
      visibility: hidden;
    }
  }
}
```